lib/Ponder/Ponder.cs

```diff
@@ -0,0 +1,70 @@
+using System;
```

Comment:
This file originally was only on lib folder on Rebus project. I added to the Rebus.MongoDb project to avoid having file outside of the folder project, but forget to remove this duplicated file. So, i remove this file and let only Ponder.cs which is in the ReBus.MongoDb project folder.